website/docs/getting-started/setup/instance-configuration/custom-domain/README.md

```diff
@@ -1,95 +1,75 @@
-
-:::info
-Please ensure port 80 on your server is open and accessible from the Internet for the HTTPS certificate to be provisioned.
-:::
-
-## Heroku
+* Rename the certificate file as `fullchain.pem` and key file as `privkey.pem`.
+* Copy these files into the subdirectory `<MOUNTING-DIRECTORY>/ssl/`. Ensure that you change `<MOUNTING-DIRECTORY>` by the mounting volume directory available in the `docker-compose.yml`. For example, the default value is `./stacks`.
+* Add your Custom Domain to the environment variable `APPSMITH_CUSTOM_DOMAIN` in the 'docker.env' file.
+* Restart the container using `docker restart appsmith`
 
-1. Go to the **Settings** tab in your Heroku app
-2. Click the `Add domain` button in the `Domains` section
-3. Input your domain name & click `Next`. Heroku provides you with a DNS target that you can map your domain with.
-4. Go to your DNS provider and make sure that your custom DNS Record (Ex. **appsmith.yourcompany.com**) is updated to map to the `DNS Target`
+Provided that the certificate files are present in the `<MOUNTING-DIRECTORY>/ssl` folder, the container uses them.
 
 :::note
-* Once you use a custom domain, You might want to set up SSL for your dyno. Please check the official document of Heroku [how to configure SSL](https://devcenter.heroku.com/articles/ssl)
-* The dyno needs to be upgraded to at least "**Hobby"** type to use this feature of Heroku
+If you want to configure a Custom SSL on your Heroku or DigitalOcean Appsmith installation platform, follow the steps as listed in the guides below:
+* [How to Configure SSL for your Heroku dyno](https://devcenter.heroku.com/articles/ssl)
+* [How do I install an SSL Certificate on a DigitalOcean Droplet](https://docs.digitalocean.com/support/how-do-i-install-an-ssl-certificate-on-a-droplet/)
 :::
 
-## DigitalOcean
+## Troubleshooting
 
-To host the Appsmith DigitalOcean droplet on a custom domain, you'll need to select the Add a domain option from the dashboard.
-
-![](/img/custom\_domain.jpeg)
-
-It redirects you to a new page. Add your domain name there. Once that's done, it'll give you records of the name servers. Copy the details of the NS (name servers). Use the custom name server's configuration on your domain provider. Sometimes, it might take up to 24 or 48 hours for this to go live.
-
-:::tip
-Your Appsmith instance should be available at [https://appsmith.mydomain.com](https://appsmith.mydomain.com) with automatic certificate provisioning and renewals.
-:::
+If you’re having issues accessing Appsmith after Appsmith SSL Configuration, please see the [Unable to Access Appsmith](/help-and-support/troubleshooting-guide/deployment-errors#unable-to-access-appsmith) troubleshooting guide. If you continue to have problems reach out on [Discord Server](https://discord.com/invite/rBTTVJp) or [send an email to support](mailto:support@appsmith.com) or ask questions on the [community forum](https://community.appsmith.com/).
```

Comment:
make sure troubleshooting link is specific to custom domain or SSL issues

Reply:
I confirmed with @sum35h that this link works for a custom domain and SSL issue fix.